transformer_engine/pytorch/module/linear.py

```diff
@@ -82,12 +82,10 @@ def forward(
         ub_overlap_rs: bool,
         ub_overlap_ag: bool,
         ub_name: str,
-        is_first_module_in_mha: bool,
+        fp8_output: bool,
         fsdp_group: Union[dist_group_type, None],
     ) -> torch.Tensor:
         is_input_fp8 = isinstance(inp, Float8Tensor)
-        if is_input_fp8:
-            fp8_meta["scaling_fwd"].scale_inv[tex.FP8FwdTensors.GEMM1_INPUT] = inp._scale_inv[0]
```

Comment:
Remove the copy of `scale_inv` since it's unnecessary after https://github.com/NVIDIA/TransformerEngine/pull/1083.